NEWS.md

```diff
@@ -1,13 +1,19 @@
 ## rtables 0.6.9.9002
+### Features
+ * Modified `reorder_split_levels()` to cover more edge cases and be more stringent in the allowed inputs.
```

Comment:
Also, this can cause issues downstream. The original method did not cover some relevant situations when adding labels. Now, it raises errors when erroneously provided.